src/lua.rs

```diff
@@ -557,6 +557,14 @@ impl Lua {
         }
     }
 
+    /// Loads the Lua debug library.
+    ///
+    /// The debug library is very unsound, loading it and using it breaks all
+    /// the guarantees of rlua.
+    pub unsafe fn load_debug(&self) {
+        ffi::luaL_requiref(self.state, cstr!("debug"), ffi::luaopen_debug, 1);
```

Comment:
You need to add a `ffi::lua_pop(state, 1);` after this, and (I think) check that the stack still has room for the pushed module.

Reply:
How would I check that hte stack still has room for the pushed module?

Reply:
Fixed pop issue

Reply:
I think the correct way to do that in this case is to call `util::check_stack`

Reply:
Added the check before the module loading